### ISSUE TITLE
Option to Disable Auto-Generated Commit Messages

### ISSUE THREAD

**Issue by MitchTalmadge** [microsoft/vscode-copilot-release#1323]
I love most of GitHub Copilot, but I find the auto-generated commit messages very distracting and I would like the option to turn them off. I have never benefitted from the messages it generates, and sometimes find it frustrating.

For example, I will finish typing my message, go to click "Commit", and then right as I am hovering over the button, Copilot will generate a message and force the button to be below my cursor.

If I ever want to generate a commit message, I can use the sparkle button to the right of the text field.

Thank you!

<img width="413" alt="image" src="https://github.com/microsoft/vscode-copilot-release/assets/8848080/c10391ae-2d7a-41c9-886a-70571357074f">

**Comment by MitchTalmadge** [comment]
@lszomoru, yes, here you are:

VS Code:
```
Version: 1.90.2 (Universal)
Commit: 5437499feb04f7a586f677b155b039bc2b3669eb
Date: 2024-06-18T22:37:41.291Z
Electron: 29.4.0
ElectronBuildId: 9728852
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Darwin arm64 23.5.0
```

Copilot:
```
Id: GitHub.copilot
Version: 1.206.0
```

Copilot Chat:
```
Id: GitHub.copilot-chat
Version: 0.16.1
```

Thank you!

**Comment by lszomoru** [comment]
@MitchTalmadge, thanks for getting back to me on this matter. Could you please check the value of the `github.copilot.enable` setting? Does it contain an entry to enable Copilot completion for the `scminput` language? If so you should set that to `false`. Thanks!

**Comment by MitchTalmadge** [comment]
@lszomoru Thanks for the help. I tried changing that value to `false` and it did not disable the suggestions, even after restarting VSCode. 

![image](https://github.com/microsoft/vscode-copilot-release/assets/8848080/0ca87c12-59d8-4623-a437-a357b9822f45)

I even created a new VSCode profile with no extensions other than Copilot, and it still has this behavior when `scminput` is `false`. 

The only change I have found which disables this entirely is setting `github.copilot.editor.enableAutoCompletions` to `false`, which of course means I don't get any suggestions in my editor, so I don't want to do that.

Thanks!
